client/reader/list-stream/empty.tsx

```diff
@@ -0,0 +1,53 @@
+import DOMPurify from 'dompurify';
+import { useTranslate } from 'i18n-calypso';
+import EmptyContent from 'calypso/components/empty-content';
+import { recordAction, recordGaEvent } from 'calypso/reader/stats';
+import { useDispatch } from 'calypso/state';
+import { recordReaderTracksEvent } from 'calypso/state/reader/analytics/actions';
+
+export default function ListEmptyContent(): JSX.Element {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const queryParams = new URLSearchParams( location.search );
+	const lastPageLink = DOMPurify.sanitize( queryParams.get( 'last_page' ) ?? '' );
+
+	function onClickActionBtn(): void {
+		recordAction( 'clicked_following_on_empty' );
+		recordGaEvent( 'Clicked Following on EmptyContent' );
+		dispatch( recordReaderTracksEvent( 'calypso_reader_following_on_empty_list_stream_clicked' ) );
```

Comment:
If we want to keep these actions/events around we should probably send different ones for profile. Given the names here are all "Following" (the old and under hood name for "Recent") they seem specific to selecting to load the Following feed.  Maybe we just conditionally send the same events with the word "following" replaced with something contextual here... what do we call that page we are coming from? "profile" ? "profile lists" ? "user lists" ? 

I think your original conditional was in the right direction to separate these events when this doesn't lead back to following, but if the events are useful enough to keep around we should probably send user list (or whatever we call that page) variations in the new case too.

Reply:
Removing them sounds fine to me.

Reply:
Done https://github.com/Automattic/wp-calypso/pull/98706/commits/fd70bba31c7c4202e4af4408f89749277830e748.